app/controllers/instructeurs/procedures_controller.rb

```diff
@@ -68,15 +71,16 @@ def show
       @followed_dossiers = current_instructeur
         .followed_dossiers
         .where(groupe_instructeur_id: groupe_instructeur_ids)
+        .visible_by_administration
```

Comment:
l'impression que se serait moins sujet a erreur de merge la condition initiale plutot que la reconstruire
```suggestion
        .merge(dossiers_visibles)
```